source/reference/configure-shell-settings-global.txt

```diff
@@ -0,0 +1,104 @@
+.. _configure-settings-global:
+
+===========================================================
+Configure :binary:`~bin.mongosh` Using a Configuration File
+===========================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+You can specify ``mongosh`` settings in a global configuration file.
+When you specify settings in a configuration file, those settings are
+applied at startup.
+
+Configuration File Format
+-------------------------
+
+The ``mongosh`` configuration file uses the YAML format. All options are
+under the ``mongosh`` namespace.
+
+Configuration File Location
+---------------------------
+
+The file location where ``mongosh`` looks for the configuration file
+depends on your operating system:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 10 20
+
+   * - Operating System
+     - File Location
+
+   * - Windows
+     - ``mongosh.cfg``, in the same directory as the ``mongosh.exe``
+       binary.
+   * - macOS
+     - ``mongosh`` looks for a configuration file in the following
+       directories in the order they are listed:
+
+       1. ``/usr/local/etc/mongosh.conf``
+       #. ``/opt/homebrew/etc/mongosh.conf``
+       #. ``/etc/mongosh.conf``
+
+       Once ``mongosh`` reads a configuration file in one of these
+       directories, any remaining directories in the list are not
+       checked and configuration files in those directories are ignored.
+
+   * - Linux
+     - ``/etc/mongosh.conf``
+
+Configurable Settings
+---------------------
+
+You can specify the following ``mongosh`` settings in your configuration
+file:
+
+.. include:: /includes/list-table-shell-properties.rst
+
+Example
+-------
+
+The following configuration file sets:
+
+- ``displayBatchSize`` to 50
+- ``inspectDepth`` to 20
+- ``redactHistory`` to ``remove-redact``
+
+.. code-block:: yaml
+
+   mongosh:
+     displayBatchSize: 50
+     inspectDepth: 20
+     redactHistory: "remove-redact"
+
+Behavior with ``config`` API
+----------------------------
+
+Settings set with the :ref:`config API <configure-settings-api>`
```

Comment:
[suggestion]

I found this a bit trick to parse. Consider "Settings specified with the ...."